### DISCUSSION TITLE
Does boyfriend have hallucinations about the lemon demon?

### DISCUSSION THREAD

**Discussion by KaidenOnYT** [FunkinCrew/Funkin#1560] [Q&A] [answered]
#### #1174 Hi NinjaMuffin my question was did boyfriend halation about the lemon Demon?

**Comment by KaidenOnYT** [comment]
sorry my grammar sucks

**Comment by aflacc** [comment]
why did you mention my issue

**Comment by ghost** [comment, marked as answer]
boyfriend halation

**Reply by isakube** [reply]
boyfriend halation

**Comment by bonkmaykr** [comment]
Not sure this is worthy of a Github issue, but it _is_ a question, which meets the criteria. So.

Very interesting lore bit, I must say

**Comment by aflacc** [comment]
I honestly think the lemon demon could be another demon the spooky bois trapped like  molec or whatever his name is. They leave and he starts causing havoc. Just like in Unwanted Guest

**Comment by Techno274721** [comment]
He probably does have halucinations of the lemon demon, and this would be a good mod idea someone could make as GF is also a demon....
